glommio/src/channels/shared_channel.rs

```diff
@@ -4,7 +4,7 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2020 Datadog, Inc.
 //
 //
-use crate::channels::spsc_queue::{make, Consumer, Producer};
+use crate::channels::spsc_queue::{make, BufferHalf, Consumer, Producer};
```

Comment:
I find it a bit unfortunate that we have to expose the `BufferHalf` thing to users of this, although this is all crate-local anyway so not that big of an issue.

I'll probably merge it as is

Reply:
I thought the same, but as it is only crate-local, it seems like a relatively clean solution, at it occured to me that this was the solution with the least friction.